src/supporting-data/citations/components/LiteratureCitation.tsx

```diff
@@ -222,6 +239,10 @@ export const JournalInfo: FC<JournalInfoProps> = ({
       {volume && page && ':'}
       {page}
```

Comment:
```suggestion
      {page && `{page} `}
```
I would add the page here conditionally